velox/core/QueryConfig.h

```diff
@@ -304,6 +304,13 @@ class QueryConfig {
   /// The current spark partition id.
   static constexpr const char* kSparkPartitionId = "spark.partition_id";
 
+  /// When true, establishing the result type of an arithmetic operation
+  /// happens according to Hive behavior and SQL ANSI 2011 specification, i.e.
+  /// rounding the decimal part of the result if an exact representation is not
+  /// possible. Otherwise, NULL is returned in those cases, as previously
```

Comment:
> Otherwise, NULL is returned in those cases, as previously

nit: comment suggestion: NULL is returned when the actual result cannot be represented with the calculated decimal type.